ucore/install-ucore.sh

```diff
@@ -2,6 +2,11 @@
 
 set -ouex pipefail
 
+## CONDITIONAL: install sanoid if ZFS
+if [[ "-zfs" == "${ZFS_TAG}" ]]; then
+    rpm-ostree install sanoid 
```

Comment:
Very nitpicky, but, it's how I roll... can you please delete the trailing space on this line?

Thank you!

Reply:
no issue.  agreed and corrected.